src/Markup/Avalonia.Markup.Xaml/XamlIl/Runtime/RuntimePlatformInfo.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Runtime.InteropServices;
+
+
+namespace Avalonia.Markup.Xaml.XamlIl.Runtime;
+
+public class RuntimePlatformInfo
+{
+    private static OSPlatform IOS { get; } = OSPlatform.Create("IOS");
+    
+    private static OSPlatform Android { get; } = OSPlatform.Create("ANDROID");
+    
+    private static OSPlatform Browser { get; } = OSPlatform.Create("BROWSER");
```

Comment:
I don't think it will work. "BROWSER" might be supported in wasm runtime, but "IOS" or "ANDROID" don't.
You need to use OperatingSystem.IsXXX API instead from net5

Reply:
Btw, can we reuse IRuntimePlatform.GetRuntimeInfo here? It already knows correct OS.